README.rst

```diff
@@ -18,24 +18,29 @@ This repository provides a simple "hello world" application example for `REANA
 <http://reanahub.io/>`_ reusable research data analysis plaftorm. The example
```

Comment:
`http://www.reana.io` instead of `http://reanahub.io`.  The former is more safe WRT domain names.  See the other examples.